src/main/java/com/example/botfightwebserver/player/PlayerService.java

```diff
@@ -101,6 +104,24 @@ public boolean setCurrentSubmissionIfNone(Long playerId, Long submissionId) {
         }
         return false;
     }
+
+    public Player[] pagination(int page, int size) {
+        if (page < 0) {
+            throw new IllegalArgumentException("Page index must be zero or greater");
+        }
+        if (size <= 0) {
+            throw new IllegalArgumentException("Page size must be greater than 0");
+        }
+
+        Pageable pageable = PageRequest.of(page, size);
+
+        // Fetch the players for the specified page
+        Page<Player> playerPage = playerRepository.findAll(pageable);
+
```

Comment:
I should've mentioned this, but the players and pages should be sorted by their glicko rating. So Page 1 has the {size} best players sorted in order, then page 2 and so forth.

Reply:
Added!